feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -64,9 +66,13 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
     switchMap((cartId) => this.multiCartFacade.getCartEntity(cartId))
   );
 
+  // Flag to prevent cart loading when logged in with code flow
```

Comment:
Could you add more context and explain in the comment what's the responsibility of such variable?

Reply:
Added